### PR TITLE
Add new type to support arearange graph

### PR DESCRIPTION
**Pull request by dolanmiu**
Area Range needs 3 inputs for it to `addPoint`:

http://jsfiddle.net/k8EKj/

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by jgresham — APPROVED** [review]

File: types/highcharts/index.d.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
-// Type definitions for Highcharts 5.0
+// Type definitions for Highcharts 7.1
```
Comment (rbuckton) [inline types/highcharts/index.d.ts]: When upgrading the major version, please consider the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package

**Review by rbuckton — CHANGES_REQUESTED** [review]

File: types/highcharts/index.d.ts (hunk 1 above)
Comment (baltie) [reply types/highcharts/index.d.ts]:
```suggestion
// Type definitions for Highcharts 5.0
```

Highcharts are supplying their own TypeScript definitions for Highcharts 7 and onwards. Also, this pull request does not include anything to lift the type definitions to version 7 either, so it is misleading. I suggest you leave this comment as it was.

**Comment by dolanmiu** [comment]
Done, I reverted the version back to 5.0

**Review by rbuckton — APPROVED** [review]

**Comment by rbuckton** [comment]
Thanks for the contribution!

**Pull request merged**